automerge-c/src/result.rs

```diff
@@ -168,6 +165,80 @@ impl<'a> PartialEq for AMvalue<'a> {
     }
 }
 
+impl From<&AMvalue<'_>> for Result<am::ScalarValue, am::AutomergeError> {
```

Comment:
> What's the reasoning behind implementing From rather than TryFrom here?

@alexjg, there was no reasoning, only my failure to realize that `TryFrom` is the more appropriate trait to implement here.

Reply:
@alexjg, it's `TryFrom` now.